tailwind.config.js

```diff
@@ -1,163 +1,102 @@
+const colors = require("@esri/calcite-colors/colors.json")
```

Comment:
Only the `ui-` colors have css variables attached to them that I can reference in the config file.  So I needed to import the json file to reference non-ui colors (grayscales, etc.)